docs/_documentations/mdt-che-setupregistries.md

```diff
@@ -1,25 +1,80 @@
+
+# Codewind Docker registry guidance
+When you run Codewind on Kubernetes for Codewind style projects, Codewind uses [`buildah`](https://github.com/containers/buildah) to build container images from Dockerfiles and to push the images to an image registry that you specify. Use the **Image Registry Manager** in your Codewind IDE to configure the registry to use. 
+
+Use any registry as long as `buildah` and the node that Codewind is running on can resolve the registry hostname. If `buildah` cannot resolve the registry hostname, `buildah` cannot push your projects to the registry, and deployment on Kubernetes is prevented.
+
+### Examples:
+These examples show recommended values for common registries. The following deployment registries are tested and verified with Codewind:
+- Docker Hub:
+    - Address: `docker.io`
+    - Namespace: `<namespace>`
+    - Credentials: Docker Hub user name and password or access token
+    - **Note:** For Docker Hub, the `Namespace` value is likely to be your user name. 
+- Quay.io:
+    - Address: `https://quay.io`
+    - Namespace: `<namespace>`
+    - Credentials: Quay.io user name and encrypted password
+- Artifactory
+    - Address: `<artifactory-hostname>`
+    - Namespace: `<namespace>`
+    - Credentials: Artifactory user name and access token
+- OpenShift Registry
+    - Address: `docker-registry.default.svc:5000`
+    - Namespace: `<project>`
+    - Credentials: OpenShift user name and access token
+
+### Specifying image registries for Codewind 
+Three scenarios in Codewind require you to specify which image registry is in use:
+
+- In the following scenario, configure Codewind with an image registry to push the application image to that specific image registry:
+    - Run Codewind on a remote Kubernetes cluster, including Che, to develop a Codewind style project.
+- In the following scenario, configure Codewind to use the credentials for a specific image registry:
+    - Run Codewind on a remote Kubernetes cluster, including Che, to develop an Appsody style project.
+    - The image registry for the Appsody stack requires credentials.
+- In the following scenario, from the command line, enter the `docker login` command in the registry before you create the Appsody project:
+    - Run Codewind locally to develop an Appsody style project.
+    - The image registry for the Appsody stack requires credentials. 
+
+If you do not develop any Codewind style projects, and you use an image registry that does not require credentials, then you do not need to specify an image registry.
+
+### Adding registries to OKD and OpenShift
+To use the OpenShift internal Docker registry with Codewind, see [Adding the OpenShift internal registry with Codewind](openshiftregistry.html).
 
 Next step: [Create Codewind workspace](mdt-che-createcodewindworkspace.html)
+Next step: [Using Codewind in Theia](mdt-che-usingtheia.html)
```

Comment:
Which "Next Step" link do we want? I'm thinking maybe [Adding the OpenShift internal registry with Codewind](openshiftregistry.html) might be the way to go?

Reply:
I don't know if I would put it as the next step here. It's kind of like a sibling doc to this one.